app/models/timer_session.rb

```diff
@@ -44,7 +44,13 @@ def relevant_issues
   end
 
   def overlaps?(other_session)
-    (timer_start...timer_end).overlaps?(other_session.timer_start...other_session.timer_end)
+    # Truncate seconds for overlap calculation
+    my_start = timer_start.change(sec: 0)
+    my_end = timer_end.change(sec: 0)
+    other_start = other_session.timer_start.change(sec: 0)
+    other_end = other_session.timer_end.change(sec: 0)
+
+    (my_start...my_end).overlaps?(other_start...other_end)
```

Comment:
```suggestion
    (other_session.timer_start - timer_end) < -30.seconds
```

Does this work?

Reply:
The true differential method would look like this:
```ruby
(other_session.timer_start > timer_start && (timer_end - other_session.timer_start) > 30.seconds) || (timer_start > other_session.timer_start && (other_session.timer_end - timer_start) > 30.seconds)
```

Which isn't much simpler than the `round_to_nearest_minute` solution.